Sources/App/Core/SiteURL.swift

```diff
@@ -45,8 +45,11 @@ enum SiteURL: Resourceable {
             case let .images(name):
                 return "images/\(name)"
 
-            case let .package(.value(owner), .value(repository)):
-                return "\(owner)/\(repository)"
+            case let .package(.value(owner), .value(repo)):
+                let owner = owner.addingPercentEncoding(withAllowedCharacters: .urlPathAllowed) ?? owner
+                let repo = repo.addingPercentEncoding(withAllowedCharacters: .urlPathAllowed) ?? repo
+                return "\(owner)/\(repo)"
```

Comment:
I don't think we want to fail here if `addingPercentEncoding` returns `nil`, so I'm just falling back to the original value.